### DISCUSSION TITLE
Welcome to QUICK-MARKET Discussions!

### DISCUSSION THREAD

**Discussion by Sacro10** [Sacro10/QUICK-MARKET#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Sacro10** [comment]
Ok this is the panel open for discussion

**Reply by Ryan-browny** [reply]
okay i am there

**Reply by Sacro10** [reply]
Ok that's good
So we'll work on those  parameters here

**Reply by Dikielle** [reply]
Alright

**Reply by Ryan-browny** [reply]
what are we to do now

**Reply by Dikielle** [reply]
So, what's the plan?

**Reply by Sacro10** [reply]
We are going to do the git commands on git
So lets go to Git

**Reply by Ryan-browny** [reply]
okay

**Reply by Sacro10** [reply]
Please someone should activate data lets do this in class
